Tests/scripts/infrastructure_tests/mock_unit_tests.py

```diff
@@ -0,0 +1,31 @@
+from mock import patch
+from Tests.mock_server import AMIConnection, clean_filename, get_mock_file_path, get_log_file_path, get_folder_path
+
+
+def test_clean_filename():
+    assert clean_filename(u'th)))i(s) is a (test8)8   8') == 'th___i_s__is_a__test8_8___8'
+    assert clean_filename(u'n&%ew $r#eplac@es', replace='&%$#@') == 'n__ew _r_eplac_es'
+
+
+def test_silence_output():
```

Comment:
check that nothing is returned? or maybe just don't test this

Reply:
It's not that simple... this file contains mostly placeholders for unit tests that i will develop in the coming days.